src/System.Resources.ReaderWriter/src/System/Resources/ResourceReader.cs

```diff
@@ -389,99 +462,5 @@ private int Read7BitEncodedInt()
             } while ((b & 0x80) != 0);
             return count;
         }
-
-        internal sealed class ResourceEnumerator : IDictionaryEnumerator
```

Comment:
If someone was accessing it via the interface, isn't that now going to get slower, as it'll be interface dispatch on structs vs on classes?